lib/integration/common.ts

```diff
@@ -1,4 +1,62 @@
 import { DateTime } from "luxon";
 import { TIME_ZONE } from "../../config/config";
+import { RezervoSchedule, RezervoWeekSchedule } from "../../types/rezervo";
+import { createClassPopularityIndex } from "../popularity";
 
 export const calculateMondayOffset = () => DateTime.now().setZone(TIME_ZONE).weekday - 1;
+
+export const getDateTime = (date: string): DateTime => DateTime.fromISO(date, { zone: TIME_ZONE, locale: "no" });
```

Comment:
Maybe introduce a locale variable similar to `TIME_ZONE`

Reply:
Dunzos

Reply:
Oh, I meant the locale value 'no' :upside_down_face:

Reply:
Ah, right. Fixed in https://github.com/mathiazom/sit-rezervo-web/pull/39/commits/a4b8c0076e5908dd2fc336c996f5bd9ba80b889f